README.md

```diff
@@ -1,15 +1,12 @@
+# Gatsbyn Suomenkielinen Käännös
 
-# Gatsby Finnish Translation
+Tämä repo sisältää Gatsbyn suomenkielisen käännöksen.
 
-This repo contains the Finnish translation for Gatsby.
+Siirry katsomaan [tänne](https://github.com/gatsbyjs/gatsby-fi/issues/1) käännöksen etenemistä,
+jotta voit valita sivusi ja varmistaa, että kukaan ei ole aloittanut kääntämistä.
```

Comment:
No line breaks in the middle of sentences. And some wording suggestions to make it flow more naturally. And correcting the translation progress issue reference now that we have it :)

```suggestion
[Täältä](https://github.com/gatsbyjs/gatsby-fi/issues/4) voit seurata käännöksen etenemistä ja varmistaa, ettei kukaan ole vielä aloittanut jonkin osion kääntämistä.
```